src/components/SelectionCard.js

```diff
@@ -7,18 +7,24 @@ const Body = styled.div`
   border-width: 1px;
 `;
 
-const SelectionCard = ({ unit }) => {
-  const { name, cost } = unit; //todo temporary will need to expand deez props more when the comp grows into something usable
+const SelectionCard = ({ unit, addFunction }) => {
+  const { name, point_cost } = unit; //todo temporary will need to expand deez props more when the comp grows into something usable
   return (
-    <Body>
+    <Body
+      onClick={e => {
+        e.stopPropagation();
```

Comment:
Did you actually need to stop propagation? Was there an issue you were seeing?

Reply:
Yeah basically clicking a child item here was also triggering the headers on click and collapsing >_<